sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -169,7 +169,7 @@ class Analyzer(
 
   def resolver: Resolver = conf.resolver
 
-  protected val fixedPoint = FixedPoint(maxIterations)
```

Comment:
Can you leave some comments here about why we need this change?

Reply:
@maropu Sure, added in commit 8a20a5c622d4b52980036c620d7e848925e43f38. Thanks for point it out!